credentials/src/main/kotlin/web5/sdk/credentials/model/VcDataModel.kt

```diff
@@ -107,9 +109,9 @@ public class VcDataModel(
    * Builder class for creating [VcDataModel] instances.
    */
   public class Builder {
-    private var id: URI? = null
-    private var context: MutableList<URI> = mutableListOf()
-    private var type: MutableList<String> = mutableListOf()
+    private lateinit var id: URI
```

Comment:
could u explain why lateinit was added?

Reply:
Yea it was a kotlin thing giving me compile time errors, it wont allow non nullable properties to be left uninitialized. I could be using Kotlin feature wrong? but it seems like this is the only way to use the builder this way